dashboard/src/components/IssueDetails/IssueDetails.tsx

```diff
@@ -185,6 +185,7 @@ export const IssueDetails = ({
         />
       }
     >
+      <title>{`Issue: ${data?.comment}`}</title>
```

Comment:
This might set the title to `Issue: undefined`. Check it out:

- https://www.typescriptlang.org/play/?#code/C4TwDgpgBAIghsOUC8UDeAoK2oGMD2AtoRAHbABcUAysAE4CWpA5hgL4YYBmArqbsAb5SUADb5mAFQgBnYAAoAJgjgB+KvEQBKdFhyiIwKMAgAPI6gAGASRkyeEKgBI0yxKoB0BYmWBtLetgEpDL4Bh7izPIm5lrsnJHScvJ8ihBcTBCKWkA